src/controllers/roomController.js

```diff
@@ -0,0 +1,118 @@
+import { Op } from 'sequelize';
+import { uploadPic } from '../config/cloudinary';
+import RoomServices from '../services/roomServices';
+import AccommodationServices from '../services/accommodationService';
+import Response from '../helpers/sendResponse';
+import code from '../helpers/statusCode';
+
+/** Class representing room controllers */
+export default class roomController {
+  /**
+   * create room
+   * @param {Object} req provides the requests from Admin to controllers
+   * @param {Object} res provides responses to the admin
+   * @return {object} Oject of data or error
+  */
+  static async createRoom(req, res) {
+    try {
+      const { accommodationId } = req.params;
+      const accommodationExist = await AccommodationServices.findAccommodation(
```

Comment:
@Mucyo-Aime-Christian, I suggest that this block along side it's if block can be removed and create it's own middleware to check if the accomodation exists as it is repeating it's self in this controller.